### PR TITLE
Release v1.0.3 - Encodings, README

### PR DESCRIPTION
**Pull request by BrendanThompson**
Merge of `develop` into `master` for the upcoming `v1.0.3` release.

### CODE REVIEW

**Comment by BrendanThompson** [comment]
@zenspider can you please review this, once we have merged into `master` will create a release as per #13

File: README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# rb-scpt
+
+## Overview
+Ruby AppleScript (rb-scpt) is a high-level, user-friendly Apple event bridge that allows you to control scriptable Mac OS X applications using ordinary Ruby scripts.
+
+## Contributing
+
+All contributions to this repo are welcome, infact wanted. Please ensure that all work is done off of the `develop` branch with a PR back into master. New versions of the gem will be built from master.
```
Comment (zenspider) [inline README.md]: "In fact"

**Review by zenspider — CHANGES_REQUESTED** [review]
Typo

**Comment by BrendanThompson** [comment]
@zenspider fixed the typo :)

**Review by zenspider — APPROVED** [review]

**Comment by zenspider** [comment]
DO IT!

**Pull request merged**